### PR TITLE
Teams config.name + integrity tests

### PR DESCRIPTION
**Pull request by akarve**

### CODE REVIEW

File: catalog/config.js.tmpl [hunk 1]
```diff
@@ -3,6 +3,6 @@ window.__CONFIG = {
   api: '$REGISTRY_URL',
   signOutUrl: '$SIGN_OUT_URL',
   stripeKey: '$STRIPE_KEY',
-  team: '$TEAM_ID' ? { id: '$TEAM_ID' } : undefined,
+  team: '$TEAM_ID' ? { id: '$TEAM_ID' name: '$TEAM_NAME'} : undefined,
```
Comment (meffij) [inline catalog/config.js.tmpl]: what happens if TEAM_NAME isn't defined?

Comment (akarve) [reply catalog/config.js.tmpl]: that is a non-issue. missing team.name is a soft failure `check(...error=false)`. the catalog will get an empty string. the more serious issue with this line of code is that it was missing a comma. fixed.

Comment (meffij) [reply catalog/config.js.tmpl]: lol I had one job

File: catalog/app/containers/Admin/index.js [hunk 2]
```diff
@@ -29,7 +29,7 @@ import Status from './Status';
 import Policies from './Policies';
 
 
-const teamName = config.team && config.team.name;
+const teamName = config.team ? config.team.name || config.team.id : '?';
```
Comment (meffij) [inline catalog/app/containers/Admin/index.js]: why '?'? why not empty string or undefined or something?

File: catalog/config.js.tmpl (hunk 1 above)
Comment (akarve) [reply catalog/config.js.tmpl]: still it was a good question. the catalog with fall back on `team.id` in other cases. and i actually added a case for that just now in `Admin/index.js`. The dangerous thing about this file, btw, is that we don't lint it, which I will push a fix for.

**Pull request merged**